### PR TITLE
package.json related fixes

### PR DESCRIPTION
**Pull request by slang800**
just random stuff I noticed while reading the code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,12 +97,3 @@ It's a miracle. Get it at http://www.graylog2.org/
 ## Installation
 
     npm install graylog2
-
-## License
-
-See LICENSE file. Basically, it's a kind of "do-whatever-you-want-for-free" license.
-
-## Original author
```
Comment (stelcheck) [inline README.md]: Why do you remove the original author?

File: README.md [hunk 2]
```diff
@@ -97,12 +97,3 @@ It's a miracle. Get it at http://www.graylog2.org/
 ## Installation
 
     npm install graylog2
-
-## License
```
Comment (stelcheck) [inline README.md]: Why remove this?

File: graylog.js [hunk 3]
```diff
@@ -251,7 +251,7 @@ graylog.prototype.send = function (chunk, server, cb) {
 
     this._unsentChunks += 1;
 
-    client.send(chunk, 0, chunk.length, server.port, server.host, function (err/*, bytes */) {
```
Comment (stelcheck) [inline graylog.js]: Not sure there is a point to remove this... @ronkorving mind commenting?

File: graylog.js [hunk 4]
```diff
@@ -291,8 +291,6 @@ graylog.prototype.close = function (cb) {
         });
     }
 
-    var that = this;
```
Comment (stelcheck) [inline graylog.js]: That is used below.

File: package.json [hunk 5]
```diff
@@ -1,24 +1,31 @@
 {
```
Comment (stelcheck) [inline package.json]: Whats the point of changing the indent?

File: README.md (hunk 2 above)
Comment (slang800) [inline README.md]: The MIT license isn't a "do-whatever-you-want-for-free" license... it requires attribution, and maintaining the license for any significant portion of the software that's copied. "do-whatever-you-want-for-free" is public domain dedication, or WTFPL.

Plus it's covered in package.json & LICENSE file now... so it's pretty redundant.

File: README.md (hunk 1 above)
Comment (slang800) [inline README.md]: it's not removed - it's moved into the package.json, so it's searchable.

File: graylog.js (hunk 4 above)
Comment (slang800) [inline graylog.js]: it's defined twice (see: d9fc056c9971d65c5b2dd50b4b2d35438cb7ba4f)

Comment (stelcheck) [inline graylog.js]: Ah, understood. Thanks.

File: package.json (hunk 5 above)
Comment (slang800) [inline package.json]: this part is actually enforced by npm itself - whenever you run an npm command that alters the package.json, it changes the indent back to 2 spaces (which is a style from the node.js core).

File: README.md (hunk 1 above)
Comment (stelcheck) [inline README.md]: Humans read README, not package.json. Could we keep it in both place perhaps?

File: README.md (hunk 2 above)
Comment (stelcheck) [inline README.md]: Could we link to the LICENSE file instead then? Something like 

See the (license file)[./LICENSE]

?

File: package.json (hunk 5 above)
Comment (stelcheck) [inline package.json]: Ah, didn't know that. No worries then :)

File: README.md (hunk 2 above)
Comment (slang800) [inline README.md]: hmm... I think I can do one better: ea148a7 :smile:

Comment (stelcheck) [inline README.md]: Awesome :D

File: graylog.js (hunk 3 above)
Comment (slang800) [inline graylog.js]: it's functionally identical... the only reason for getting rid of it is that it's shorter, and some code highlighters trip on it.

Comment (ronkorving) [inline graylog.js]: Code highlighters trip on it? Yikes. Anyway, I don't mind it being removed.

File: package.json [hunk 6]
```diff
@@ -1,24 +1,31 @@
...
-    },
-    "main": "./graylog.js"
+  "name": "graylog2",
+  "description": "Graylog2 client library for node.js",
+  "version": "0.1.3",
+  "author": "Marc Trudel-Belise",
+  "bugs": {
+    "url": "http://github.com/Wizcorp/node-graylog2/issues"
+  },
+  "contributors": [
```
Comment (stelcheck) [inline package.json]: Why not add yourself here?

**Comment by stelcheck** [comment]
Mostly LGTM. Just add yourself as a contributor in package.json (unless you don't want to), and we are good to go!

File: package.json (hunk 6 above)
Comment (slang800) [inline package.json]: sure!

**Comment by slang800** [comment]
all set! :smile:

**Comment by stelcheck** [comment]
Will you re-submit your other PR as well? If so, I'll wait for it and make a release once its merged.

**Pull request merged**